app/controllers/carts_controller.rb

```diff
@@ -0,0 +1,13 @@
+class CartsController < ApplicationController
+  def index
+    session[:cart] ||= []
+    dish = Dish.find(params[:dish_id])
+    @restaurant_id = params[:restaurant_id]
+    if session[:cart] << dish
+      redirect_to restaurant_menus_path(@restaurant_id)
+      flash[:notice] = "#{dish.name} added to cart"
+    else
+      flash[:notice] = "Error: #{dish.name} was NOT added to cart"
```

Comment:
Why are we implementing the sad path already when we have no tests requiring this? Stick to your tests and only implement what's required to get them to pass.